assets/airports/eidw.json

```diff
@@ -883,442 +1076,766 @@
       "end":    [["N53d26m13.16", "W6d15m43.12", "217ft"], ["N53d25m11.66", "W6d14m58.54", "202ft"]],
       "length":      2.072,
       "ils":         [true, true]
+    },
+    {
+      "name":        ["10X", "28X"],
+      "name_offset": [[1000, 1000], [1000, 1000]],
+      "end":    [["N53d25m20.75", "W6d17m24.27", "242ft"], ["N53d25m12.94", "W6d15m02.08", "202ft"]],
+      "length":      2.637,
+      "ils":         [true, true]
+    },
+    {
+      "name":        ["16X", "34X"],
+      "name_offset": [[1000, 1000], [1000, 1000]],
+      "end":    [["N53d26m13.16", "W6d15m43.12", "217ft"], ["N53d25m11.66", "W6d14m58.54", "202ft"]],
+      "length":      2.072,
+      "ils":         [true, true]
     }
   ],
   "sids": {
     "SUROX": {
       "icao": "SUROX",
       "name": "Surox",
-      "suffix": {"10":"6E", "28":"1A", "16":"5M", "34":"1G"},
+      "suffix": {"10":"6E", "28":"1A", "16":"5M", "34":"1G", "10X":"6F", "28X":"1B", "16X":"5P", "34X":"1H"},
       "rwy": {
         "10": [["DW551", "A30+"], "LEMTA", "DODIG"],
-        "28": [["OE", "A30+"], "DWW04"],
-        "16": [["DWS77", "A30+"], "BATED", "ERUDA", "DODIG"],
-        "34": [["DWN26", "A30+"], "DWW06", "DWW04"]
+        "28": [["OE", "A30+"], "_LIFFY1", "_LIFFY2", "_LIFFY3", "_LIFFY4", "_LIFFY5", "_LIFFY6", "_LIFFY7", "_LIFFY8", "_LIFFY9", "_LIFFY10", "DWW04"],
+        "16": [["DWS77", "A30+"], "_BATED1", "_BATED2", "_BATED3", "_BATED4", "_BATED5", "_BATED6", "_BATED7", "_BATED8", "_BATED9", "_BATED10", "_BATED11", "_BATED12", "_BATED13", "_BATED14", "_BATED15", "_BATED16", "_BATED17", "_BATED18", "_BATED19", "_BATED20", "_BATED21", "_BATED22", "_BATED23", "_BATED24", "_BATED25", "_BATED26", "_BATED27", "_BATED28", "_BATED29", "BATED", "ERUDA", "DODIG"],
+        "34": [["DWN26", "A30+"], "_BAL1", "_BAL2", "_BAL3", "_BAL4", "_BAL5", "_BAL6", "_BAL7", "_BAL8", "_BAL9", "_BAL10", "_BAL11", "DWW06", "DWW04"],
+        "10X": ["DW931", "LEMTA", "DODIG"],
+        "28X": ["DUB", "DWN28", "DWW04"],
+        "16X": ["OP", "BATED", "ERUDA", "DODIG"],
+        "34X": ["DUB", "DWN28", "DWW04"]
       },
       "body": ["SUROX"],
-      "draw": [["OE", "DWW04"], ["ERUDA", "DODIG", "SUROX"], ["DWW06", "DWW04", "SUROX"]]
+      "exitPoints": {
+        "ENULA": ["ENULA"]
+      },
+      "draw": [["ERUDA", "DODIG", "SUROX"], ["OP", "BATED"], ["DUB", "DWN28", "DWW04"], ["_LIFFY10", "DWW04", "SUROX"]]
```

Comment:
I don't think so. The procedure ends at `SUROX` actually, so we should amend the exit point as such.

The intent here is clearly to extend the flightplan a bit further, and that should be achieved by `AIRPORT.DEPARTURE.EXITPOINT..FIXXA..FIXXB..FIXXC`, but this is not yet possible (see #81). In the meantime, we're better off sticking with these shorter routes than overbuilding the procedures. Adjusting as such.